prime/infra/dev/prime-client-api.yaml

```diff
@@ -151,6 +151,28 @@ paths:
           description: "User not found."
       security:
       - auth0_jwt: []
```

Comment:
Also copy same changes to prime-client-api.yaml in new-dev and new-prod.

Reply:
Done! Merge with latest develop too and should build and pass tests now...